src/TLuaInterpreterMapper.cpp

```diff
@@ -286,10 +289,7 @@ int TLuaInterpreter::addAreaName(lua_State* L)
     // Note that adding an area name implicitly creates an underlying TArea instance
     lua_pushnumber(L, host.mpMap->mpRoomDB->addArea(name));
     host.mpMap->setUnsaved(__func__);
-
-    if (host.mpMap->mpMapper) {
-        host.mpMap->mpMapper->updateAreaComboBox();
-    }
+    host.mpMap->update();
```

Comment:
Adding a new area name will not cause the map to change - only the `QComboBox` that shows the areas - so I don't think this change is necessarily required.